### ISSUE TITLE
Bug: For diff with context containing empty line the empty line is not shown

### ISSUE THREAD

**Issue by PLukas2018** [otakustay/react-diff-view#214]
For diffs with context around hunks (eg. using `git diff -U1`), if the context line below a diff is empty then the line is not shown.

The bug exists probably since `v2.4.2` and is probably caused by `trim` method on the diff which removes the empty context line.
https://github.com/otakustay/react-diff-view/blob/49cebd0958ef323c830395c1a1da601560a71781/src/utils/parse.ts#L97

## [Example](https://codesandbox.io/p/sandbox/react-diff-view-bug-l8539j?file=%2Fsrc%2Findex.js%3A35%2C28&layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clsa7o0im0006286aiw7xetv9%2522%252C%2522sizes%2522%253A%255B70%252C30%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clsa7o0im0002286az0kcbud7%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clsa7o0im0003286aqs08mwle%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clsa7o0im0005286a3u7tueok%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clsa7o0im0002286az0kcbud7%2522%253A%257B%2522id%2522%253A%2522clsa7o0im0002286az0kcbud7%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clsa8pbyl0002286a64gn978j%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A10%252C%2522startColumn%2522%253A27%252C%2522endLineNumber%2522%253A10%252C%2522endColumn%2522%253A27%257D%255D%252C%2522filepath%2522%253A%2522%252Fpackage.json%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%252C%257B%2522id%2522%253A%2522clsabzmmk0002286awaz5ag3z%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522initialSelections%2522%253A%255B%257B%2522startLineNumber%2522%253A35%252C%2522startColumn%2522%253A28%252C%2522endLineNumber%2522%253A35%252C%2522endColumn%2522%253A28%257D%255D%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.js%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clsabzmmk0002286awaz5ag3z%2522%257D%252C%2522clsa7o0im0005286a3u7tueok%2522%253A%257B%2522id%2522%253A%2522clsa7o0im0005286a3u7tueok%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522id%2522%253A%2522clsacma5y00f8286aycuf3yzs%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clsacma5y00f8286aycuf3yzs%2522%257D%252C%2522clsa7o0im0003286aqs08mwle%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clsa7o0im0003286aqs08mwle%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Atrue%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D):
  
  **Older version of a file:**
  ```c
  void foo() {
  
    if (cond)
      doSomething();
    // Below is empty line
  
  }
  ```
  
 **New version of the file:**
  ```c
  void foo() {
  
    if (cond) {
      doSomething();
      doSomethingNext();
    }
    // Below is still empty line
  
  }
  ```
  
  **Diff gained by `git diff -U1`**
  ```patch
  diff --git a/a.c b/a.c
  index ca1fe9c..e6f7947 100644
  --- a/a.c
  +++ b/a.c
  @@ -2,5 +2,7 @@ void foo() {
   
  -    if (cond)
  +    if (cond) {
           doSomething();
  -    // Below is empty line
  +        doSomethingNext();
  +    }
  +    // Below is still empty line
   
  ```
  
  **Result of visualisaton using react-diff-view:** (with missing context line below diff)
  ![image](https://github.com/otakustay/react-diff-view/assets/80856731/ab7e88ca-141b-4a04-a707-1fea74a0ca7e)
  
  
  **[Expected result (`v2.4.1`)](https://codesandbox.io/p/sandbox/react-diff-view-old-without-bug-ghnflk?file=%2Fsrc%2Findex.js%3A32%2C2):** (with context line below diff)
  
  ![image](https://github.com/otakustay/react-diff-view/assets/80856731/706bcf64-cdfe-46a3-afec-faeede997ea2)

**Comment by otakustay** [comment]
Seems you are working on it, I'll wait for a PR, thanks

**Comment by PLukas2018** [comment]
Solved by #216

**Comment by PLukas2018** [comment]
Solved by https://github.com/otakustay/react-diff-view/pull/216
